thingif/src/main/java/com/kii/thingif/query/GroupedHistoryStatesQuery.java

```diff
@@ -70,7 +70,7 @@ public Builder setClause(@Nullable QueryClause clause) {
 
         /**
          * Set firmware version for grouped query.
-         * @param firmwareVersion firmware version
+         * @param firmwareVersion older firmware version, which defined this alias.
```

Comment:
firmwareVersion to validate the alias in case the given alias is old and does not belong to the current thingType+firmware version assigned to the target thing.